src/OpenLoco/src/Windows/Construction/Construction.h

```diff
@@ -24,82 +24,109 @@ namespace OpenLoco::Ui::Windows::Construction
+        uint8_t signalGhostTrackId;                     // 0x0113604E
+        uint8_t signalGhostTileIndex;                   // 0x0113604F
+        uint8_t pad_1136050[4];                         // 0x01136050
+        uint8_t modList[4];                             // 0x01136054
+        uint8_t modGhostRotation;                       // 0x01136058
+        uint8_t modGhostTrackId;                        // 0x01136059
+        uint8_t modGhostTileIndex;                      // 0x0113605A
+        uint8_t pad_113605B[2];                         // 0x0113605B
+        uint8_t makeJunction;                           // 0x0113605D
+        uint8_t pad_113605E[3];                         // 0x0113605E
+        uint8_t constructionHover;                      // 0x01136061
+        uint8_t trackType;                              // 0x01136062
+        uint8_t byte_1136063;                           // 0x01136063
+        uint8_t constructionRotation;                   // 0x01136064
+        uint8_t byte_1136065;                           // 0x01136065
+        uint8_t constructionArrowFrameNum;              // 0x01136066
+        uint8_t lastSelectedTrackPiece;                 // 0x01136067
+        uint8_t lastSelectedTrackGradient;              // 0x01136068
+        uint8_t ghostRemovalTrackRotation;              // 0x01136069
+        uint8_t ghostRemovalTrackId;                    // 0x0113606A
+        uint8_t stationGhostRotation;                   // 0x0113606B
+        uint8_t stationGhostTrackId;                    // 0x0113606C
+        uint8_t stationGhostTileIndex;                  // 0x0113606D
+        uint8_t lastSelectedTrackModSection;            // 0x0113606E
+        uint8_t pad_113606F[3];                         // 0x0113606F
+        uint8_t byte_1136072;                           // 0x01136072
+        uint8_t byte_1136073;                           // 0x01136073
+        uint8_t pad_1136074;                            // 0x01136074
+        uint8_t byte_1136075;                           // 0x01136075
+        uint8_t byte_1136076;                           // 0x01136076
+        uint8_t byte_1136077;                           // 0x01136077
+        uint8_t byte_1136078;                           // 0x01136078
+        uint8_t lastSelectedTrackPieceId;               // 0x01136079
+        uint8_t pad_113607A[4];                         // 0x0113607A
+        uint8_t byte_113607E;                           // 0x0113607E
+        uint8_t pad_113607F[10];                        // 0x0113607F
+        uint8_t stationGhostTypeDockAirport;            // 0x01136089
+        uint8_t pad_113608A[6];                         // 0x0113608A
+        World::TileElement backupTileElement;           // 0x01136090
+    };
+#pragma pack(pop)
+
+    static_assert(sizeof(ConstructionState) == 0x01136090 + 8 - 0x01135F3E);
+    static loco_global<ConstructionState, 0x01135F3E> _cState;
```

Comment:
Do we actually still need this to be interop? I think all the Ui code should be covered. Kinda doubt  any AI code would mess with the Ui state.

Reply:
I think you're right. I was thinking there's some model code that messes with these variables outside the UI code, but I may have been thinking of OpenRCT2. I'd prefer not to risk it for this PR, tbh. It would require some very thorough testing.

Reply:
Fair, not a big deal.